src/commands.rs

```diff
@@ -89,6 +89,10 @@ pub struct ListArgs {
     /// In the output, which name to include
     #[clap(arg_enum, long="style", default_value_t=NameStyle::ShortName)]
     name_style: NameStyle,
+
+    /// In the output, how many spaces for padding between chords
+    #[clap(short, long="padding", default_value_t=4)]
```

Comment:
I think `long="padding"` is probably not needed because it is the same as the key word below. 

```
#[clap(short, long, default_value_t=4)]
padding: u8,
```